docs/source/environments/braintumor.rst

```diff
@@ -0,0 +1,163 @@
+.. _braintumor:
+
+
+Brain Tumor PDE
+================
+
+This documentation describes the Brain Tumor DPR (Diffusion-Proliferation-Radiation) Environment, which models the growth dynamics of glioblastomas–a fast-growing type of brain cancer–and their response to external beam radiation therapy (XRT). The model is governed by a partial differential equation (PDE) that includes three key terms: diffusion of tumor cells, proliferation, and radiation-induced cell death.
+
+While tumor growth naturally occurs in three spatial dimensions and time, the model assumes spherical symmetry, allowing the simulation to be reduced to a one-dimensional radial domain over time. To further ease computation, the PDE is nondimensionalized, rescaling spatial, temporal, and concentration variables into a unitless form. As a result, the tumor’s evolving state can be represented as a one-dimensional spatial array indexed by time.
+
+The numerical methods and modeling framework are based on the methods proposed in [1]_ and [2]_.
```

Comment:
Move this to where the dimensional model is discussed. Say something like,

To build our environment, we employ the models developed in [1] and [2] which have been the standard in Glioblastoma literature after Dimensional Model header